pytest_fixtures/rh_cloud.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from broker.broker import VMBroker
+from nailgun import entities
+
+from robottelo import manifests
+from robottelo.api.utils import upload_manifest
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.hosts import ContentHost
+
+
+@pytest.fixture(scope='module')
+def organization_ak_setup(module_org):
+    """A module-level fixture to create Organization and Activation key for tests"""
+    with manifests.clone() as manifest:
+        upload_manifest(module_org.id, manifest.content)
+    ak = entities.ActivationKey(
+        content_view=module_org.default_content_view,
+        organization=module_org,
+        environment=entities.LifecycleEnvironment(id=module_org.library.id),
+        auto_attach=True,
+    ).create()
+    subscription = entities.Subscription(organization=module_org).search(
+        query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'}
+    )[0]
+    ak.add_subscriptions(data={'quantity': 10, 'subscription_id': subscription.id})
+    return module_org, ak
+
+
+@pytest.fixture(scope='module')
+def content_hosts():
+    """A module-level fixture that provides two content hosts object based on the rhel7 nick"""
+    with VMBroker(nick='rhel7', host_classes={'host': ContentHost}, _count=2) as hosts:
+        hosts[0].set_infrastructure_type('physical')
+        yield hosts
+
+
+@pytest.fixture(scope='module')
+def registered_hosts(organization_ak_setup, content_hosts):
+    """Fixture that register content hosts to Satellite."""
+    org, ak = organization_ak_setup
+    for vm in content_hosts:
+        vm.install_katello_ca()
+        vm.register_contenthost(org.label, ak.name)
+        assert vm.subscribed
+    return content_hosts
```

Comment:
I think it makes more sense to have these in the broker fixtures file

Reply:
Jake, updated the pr.